build.cake

```diff
@@ -12,14 +12,20 @@ var nugetApiKey = EnvironmentVariable("NUGET_API_KEY");
 var nugetApiUrl = EnvironmentVariable("NUGET_API_URL");
 var psNugetApiKey = EnvironmentVariable("PS_NUGET_API_KEY");
 
-DirectoryPath artifactsDir = Directory("./artifacts/");
+var artifactsDir = Directory("./artifacts/");
 var testResultsDir = artifactsDir.Combine("test-results");
 var psModuleDir = artifactsDir.Combine("Grok");
 var testCoverageOutputFilePath = testResultsDir.CombineWithFilePath("OpenCover.xml");
 
 var projectFileMain = File("./src/Grok.Net/Grok.Net.csproj");
 var projectFilePowerShell = File("./src/Grok.Net.PowerShell/Grok.Net.PowerShell.csproj");
 
+var commonSettings = new DotNetBuildSettings
```

Comment:
It'd be better to rename it to `buildSettings` to make it consistent with `testSettings`.

But what's the reason to make it a global variable instead of keeping it in the `Build` task. Only the `Build` task uses it!

There is `testSettings` which is the same but you didn't move it...
Could you explain pls

Reply:
Thanks for your feedback, Marusyk. I've updated the variable name to `buildSettings` to maintain consistency with `testSettings`, as you suggested. Regarding your question about making it a global variable versus keeping it within the Build task, I initially considered potential future use of `commonSettings` across multiple tasks, hence the decision to define it globally. However, I see your point about encapsulation and clarity. Upon reflection, I've moved both `buildSettings` and `testSettings` into their respective tasks for better scoping and organization. Let me know if you have any further suggestions or concerns.

Reply:
no other concerns at the time